### DISCUSSION TITLE
Stuttering when mouse curson on viewport

### DISCUSSION THREAD

**Discussion by dangreb** [microdee/OpenUnrealSpaceMouse#86] [General]
So i just installed the plugin for the UE 5.1, but i'm having this annoying issue.

If i have the mouse cursor outside the viewport, say all the way up to the menus, navigation goes smooth and fine. But if i have the cursor on the viewport, just stationary in there, navigation starts to stutter whenever i make a pitch movement. 

What happens exactly is that when i pitch forward, it translates the camera forward harshly, kinda like snapping to relatively long increpemnts, and when o pitch up, the same but backwards.

Anyone knows if this is this something i should fix in the settings, or just an issue with the plugin?

**Comment by microdee** [comment]
This has been raised many times, here are couple:

* https://github.com/microdee/UE4-SpaceMouse#known-nuances
* https://github.com/microdee/UE4-SpaceMouse/issues/85
* https://github.com/microdee/UE4-SpaceMouse/issues/12
